mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -486,6 +486,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 1 x
 		ARG 3 y
 		ARG 5 z
+	METHOD method_43688 asChatMessageSender ()Lnet/minecraft/class_7436;
```

Comment:
Maybe just asMessageSender if they can send game info or other message types

Reply:
The class should be renamed then.